etl/steps/data/grapher/climate/2023-12-20/surface_temperature_anomalies.py

```diff
@@ -0,0 +1,53 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("surface_temperature")
+    tb = ds_garden["surface_temperature"].reset_index()
+    #
+    # Process data.
+    #
+    tb = tb.drop(columns=["time"])
+
+    # Transpose the DataFrame to have a country column, a column identifying the measure, and year columns
+    df_melted = tb.melt(id_vars=["country", "year", "month"], value_vars=["temperature_anomaly"])
+
+    df_melted["year"] = df_melted["year"].astype(str)
+    df_melted = df_melted.rename(columns={"value": "temperature_anomaly"})
+    df_pivot = df_melted.pivot_table(
```

Comment:
Use `pivot` instead of `pivot_table`.

Reply:
By the way, our `pivot` function has a `join_column_levels_with` which you may find useful, instead of having to rename indexes.